src/platforms/javascript/index.mdx

```diff
@@ -33,56 +33,57 @@ $ npm install --save @sentry/browser
   crossorigin="anonymous"
 ></script>
 ```
+<Alert>
+<markdown>
 
-**Use of Defer**
+Using `defer`
 
-It's possible to include `defer` in your script tag, but keep in mind that any errors which occur in scripts that execute before the browser SDK script executes won’t be caught (because the SDK won’t be initialized yet). We strongly recommend that if you use `defer`, you a) place the script tag for the browser SDK first, and b) mark it, and all of your other scripts, `defer` (but not `async`), thereby guaranteeing that it’s executed before any of the others.
+If you include `defer` in your script tag, any errors that occur in scripts executed before the browser SDK script executes won’t be caught, since the SDK won’t yet be initialized. We strongly recommend that if you use `defer`, you both place the script tag for the browser SDK first and also mark it -- and all of your other scripts -- `defer` (but not `async`), guaranteeing that it’s executed before any of the others.
 
-**Note:** If you have Content Security Policy (CSP) set up on your site, you will need to add the `script-src` of wherever you're loading the SDK from, and the origin of your DSN. For example:
+</markdown>
+</Alert>
 
-- `script-src: https://browser.sentry-cdn.com`
-- `connect-src: *.sentry.io`
+**Note:** If you have Content Security Policy (CSP) set up on your site, please see [our documentation about adding to the `script-src`](/platforms/javascript/common/troubleshooting/).
 
-## Configure
+## Configure and Verify
 
-`init` Sentry as soon as possible:
+`init` Sentry as soon as possible and include the verification of your setup with an intentionally sent error that breaks your application. For example, in JavaScript, calling an undefined function throws an exception::
```

Comment:
```suggestion
`init` Sentry as soon as possible in your app, to make sure it can monitor everything that follows it. Once this is done, all unhandled exceptions are automatically captured by Sentry.

The snippet below includes an intentional error, so you can test that everything is working as soon as you set it up. To see the recorded error, log into sentry.io and open your project. Clicking on the error's title will open a page where you can see detailed information and mark it as resolved.
```